templates/page.apps.php

```diff
@@ -39,6 +39,6 @@
 			<li><?php p($l->t('Many others like a music player, a password manager, a kanban app, a download manager or a markdown editor')); ?></li>
 
 		</ul>
-		<p class="details-link"><a href="/index.php/settings/apps" target="_blank"><?php p($l->t('Browse the app store')); ?></a></p>
+		<p class="details-link"><a href="index.php/settings/apps" target="_blank"><?php p($l->t('Browse the app store')); ?></a></p>
```

Comment:
Nice finding. However we should use the https://docs.nextcloud.com/server/14/developer_manual/api/OCP/IURLGenerator.html#OCP\IURLGenerator::linkToRoute method to not hardcode any url there.

Reply:
Since there was no response so far I added a commit to the branch.